### PR TITLE
eludris: init at 0.3.3

### PR DESCRIPTION
**Pull request by ooliver1**
###### Description of changes

<https://github.com/eludris/eludris/tree/main/cli>

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -11207,6 +11207,15 @@
     github = "ony";
     githubId = 11265;
   };
+  ooliver1 = {
+    name = "Oliver Wilkes";
+    email = "oliverwilkes2006@icloud.com";
+    github = "ooliver1";
+    githubId = 34910574;
+    key = [{
```
Comment (kilianar) [inline maintainers/maintainer-list.nix:11215]:
```suggestion
    keys = [{
```

**Review by kilianar — COMMENTED** [review]
At the moment, there appears to be a merge conflict. Other than that, it seems satisfactory to me. You may want to consider utilizing `hash` and `cargoHash` in lieu of `sha256` and `cargoSha256`.

**Comment by kilianar** [comment]
I guess you need something like
```nix
eludris = callPackage ../tools/misc/eludris {
  inherit (darwin.apple_sdk.frameworks) Security;
};
```
in `pkgs/top-level/all-packages.nix`.

**Comment by ooliver1** [comment]
Thank you! I cannot test Darwin right now but hopefully this resolves it.

**Comment by ooliver1** [comment]
@kilianar Any update on the Darwin test?

**Comment by RGBCube** [comment]
Any updates on the darwin test @kilianar ?

**Comment by kilianar** [comment]
Apologies for the delayed response; I somehow overlooked it. The failure was flagged by the automated CI test conducted by OfBorg. Fortunately, all the tests are now passing, suggesting that we are ready to proceed.

**Comment by RGBCube** [comment]
Does that mean this can be merged? It would be great 😃

**Comment by kilianar** [comment]
Currently, there are merge conflicts that need to be resolved. Afterward, it would be advisable to post this pull request on Discourse to allow someone with merge access to review it.

**Pull request merged**